app/models/course.rb

```diff
@@ -121,6 +121,12 @@ def new_draft_attributes
     end.order(:changed_at, :id)
   end
 
+  scope :not_new, -> do
+    includes(site_statuses: %i[site course])
+      .where
+      .not(SiteStatus.table_name => { status: SiteStatus.statuses[:new_status] })
```

Comment:
```suggestion
      .not(SiteStatus.table_name => { status: :new_status })
```

Can it not be like that?

Reply:
https://stackoverflow.com/questions/36487900/rails-using-enum-in-a-where-query